README.md

```diff
@@ -74,23 +73,29 @@ This class diagram shows the relationships between our DAO, our service, and the
 
 This is what we plan to export to another app 
 
->{ 
->  "type" : "object", 
->  "properties" : { 
->    "latitude" : { 
->      "type" : "string" 
->    }, 
->    "longitude" : { 
->      "type" : "string" 
->    } 
->  } 
->} 
+>{
+>  "type" : "object",
+>  "properties" : {
+>    "cacheID" : {
+>      "type" : "int"
+>    },
+>    "latitude" : {
+>      "type" : "string"
+>    },
+>    "longitude" : {
+>      "type" : "string"
+>    }
+>  "description" : {
+>      "type" : "string"
+>    }
+>}
+
 
   
 
 ## Github Project Link 
 
-https://github.com/brennerdanae/CachingInApp 
+[CachingInApp](https://github.com/brennerdanae/CachingInApp)
 
```

Comment:
Added Visual link change (optional)

Reply:
Yeah... this is better markdown format... and more accessible-friendly.